airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostgresSourceDatatypeTest.java

```diff
@@ -63,7 +63,7 @@ protected Database setupDatabase() throws SQLException {
       ctx.execute("SET lc_monetary TO 'en_US.utf8';");
       // Set up a fixed timezone here so that timetz and timestamptz always have the same time zone
       // wherever the tests are running on.
-      ctx.execute("SET TIMEZONE TO 'America/Los_Angeles'");
+      ctx.execute("SET TIMEZONE TO 'America/Phoenix'");
```

Comment:
While this timezone (right now) doesn't use any DST, I'd suggest we're trying to stay away from geographical time zones altogether here, since they might change DST later on, and instead just using `UTC` or (depending on whether the mock DB containers we're using supporting this), setting a fixed offset (which would be supported by postgres: https://kb.objectrocket.com/postgresql/postgresql-set-time-zone-1064#set+the+postgres+time+zone+with+an+integer)

Reply:
Good point. Thanks Tim! I'll look into the fixed offset option. I think there may be yet another option of using "PST" or "EST" which will avoid switching to DST timezones (i.e. PST switching to PDT).

Reply:
done